app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= stylesheet_link_tag 'recipe_foods/new', media: 'all' %>
+<div class="recipe-food__add">
+  <h2 class="recipe-food__title">Add new ingredient</h2>
```

Comment:
Thanks for info, I just realized It's not working, It was working before. I will see what it the problem 👍

Reply:
For the next code review, if you see this comment. Please make sure to write the right data, Don't right `oziegjozeigjzejgez` in quantity or price because it will not be saved (We are validating price and quantity to be only numbers). That was one of the issues which prevented ingredients to be saved